src/validator/test/FetchVerticesTest.cpp

```diff
@@ -241,6 +241,80 @@ TEST_F(FetchVerticesValidatorTest, FetchVerticesProp) {
         auto result = Eq(qctx->plan()->root(), gv);
         ASSERT_TRUE(result.ok()) << result;
     }
+    // ON * with yield
+    {
+        auto qctx = getQCtx("FETCH PROP ON * \"1\", \"2\" YIELD person.name");
```

Comment:
I think the syntax here is contradictory. we can make it more meaningful.
such as
```
FETCH PROP ON person  \"1\", \"2\" YIELD name
```

Reply:
This PR keep same with 1.0. We could consider it in later.